deploy/basic/terraform/scripts/node.sh

```diff
@@ -52,8 +52,9 @@ MUSHOP_APP_URI=$(curl -L http://169.254.169.254/opc/v1/instance/metadata | jq -j
 WALLET_URI=$(curl -L http://169.254.169.254/opc/v1/instance/metadata | jq -j ".wallet_par")
 
 # get artifacts from object storage
-get_object /root/wallet.zip $${WALLET_URI}
+get_object /root/wallet.64 $${WALLET_URI}
 # Setup ATP wallet files
+base64 --decode /root/wallet.64 >  /root/wallet.zip
```

Comment:
nit (sorry :)) you have an extra space after `>`

Reply:
Will fix